### PR TITLE
Doc/9438 tkd feature checklist updates

### PR DESCRIPTION
**Pull request by TKDickson**
## Description of Change

Closes #9438 . Intended to make that checklist more explicit (both in terms of tasks, and also in terms of marking that you have done, or considered, a thing)

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

**Review by theodur — APPROVED** [review]

**Comment by TKDickson** [comment]
from our qa sync meeting - standardize spelling of accessibility (remove a11y), and Tom and Rachael said they'd leave comments with their other thoughts on the PR

File: .github/ISSUE_TEMPLATE/QA_feature_signoff.md [hunk 1]
```diff
@@ -5,27 +5,53 @@ title: QA Feature Signoff
...
+### Always required
+| Category | Task (plus links) | QA signoff |
+| -- | -- | -- |
+| Standards | Test plan was peer reviewed |  |
+| Standards | All tickets found during testing have been prioritized | |
+| Standards | Feature testing summary written and reviewed with feature team | |  
+| Testing | ACs met on all implementation tickets | |
+| Testing | Accessibility testing per a11y design annotations | |
+| Testing | [API errors](https://dsvavsp.testrail.io/index.php?/suites/view/92&group_by=cases:section_id&group_order=desc&display_deleted_cases=0&group_id=8943) handled well | |
+| Testing | New screens respect availability framework | |
```
Comment (rbontrager) [inline .github/ISSUE_TEMPLATE/QA_feature_signoff.md:22]: I might put new screens/changed existing screens respect AF (or something similar).

File: .github/ISSUE_TEMPLATE/QA_feature_signoff.md [hunk 2]
```diff
@@ -5,27 +5,53 @@ title: QA Feature Signoff
...
-- [ ] Accessibility testing has been performed
-- [ ] All bugs found during testing have been prioritized
-- [ ] MVP-necessary bug fixes tested
-- [ ] Finished "just before merge to develop" testing - including feature happy path and merge conflict resolution spot-testing 
+### Always required
+| Category | Task (plus links) | QA signoff |
+| -- | -- | -- |
+| Standards | Test plan was peer reviewed |  |
+| Standards | All tickets found during testing have been prioritized | |
+| Standards | Feature testing summary written and reviewed with feature team | |
```
Comment (rbontrager) [inline .github/ISSUE_TEMPLATE/QA_feature_signoff.md:18]: I feel like it might be helpful here to link to the example you have.

Comment (TKDickson) [reply .github/ISSUE_TEMPLATE/QA_feature_signoff.md]: Done. Eventually I'm planning on linking to the process documentation (will be created in 9161) for that, so added an AC there to update this link when I make it.

**Review by alexandec — APPROVED** [review]

**Pull request merged**